public/app/features/explore/TraceView/components/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -236,6 +236,10 @@ export default function SpanDetail(props: SpanDetailProps) {
   }
 
   const focusSpanLink = createFocusSpanLink(traceID, spanID);
+  // focusSpanLink is destructured directly into the dom, so remove variable that is not used or expected
+  if (focusSpanLink) {
+    delete focusSpanLink.variableMap;
+  }
```

Comment:
```suggestion
   const { variabelMap, ...spanLink } = focusSpanLink;
```
Then you can spread the spanLink which will be everything else except the `variableMap` value. It might be ok to delete the `variabelMap` in this case because it is an optional value on the type but in general it is a bad idea to mutate objects this way if they would be reused on other places.

Reply:
With some previous changes, this is no longer needed.